src/pages/patientView/styles.scss

```diff
@@ -1,43 +1,46 @@
-.patientPageHeader {
-
-  position:relative;
-  padding:20px 20px 20px 94px;
-  border-radius:5px;
-  background: #eee;
-  margin-bottom: 20px;
-
-  td {
-    vertical-align:top;
-    padding-left:10px;
-    padding-bottom:4px;
-  }
+.patientViewPage {
 
+  .patientPageHeader {
 
-  .patientIcon {
-    position:absolute;
-    top:17px;
-    left:20px;
-    font-size:66px;
-  }
+    position: relative;
+    padding: 20px 20px 20px 94px;
+    border-radius: 5px;
+    background: #eee;
+    margin-bottom: 20px;
+
+    td {
+      vertical-align: top;
+      padding-left: 10px;
+      padding-bottom: 4px;
+    }
 
-  .patientSamples {
+    .patientIcon {
```

Comment:
this stuff probably belongs in a css module associated with the patient header.  Also the patient header should be a component.